addons/netfox/rollback/rollback-synchronizer.gd

```diff
@@ -23,12 +23,12 @@ var _auth_state_property_entries: Array[PropertyEntry] = []
 var _auth_input_property_entries: Array[PropertyEntry] = []
 var _nodes: Array[Node] = []
 
-var _states: Dictionary = {} #<tick, Dictionary<String, Variant>>
-var _inputs: Dictionary = {} #<tick, Dictionary<String, Variant>>
-var _latest_state_tick: int = -1
+var _states: Dictionary = {}
+var _inputs: Dictionary = {}
+var _latest_state: int = -1
```

Comment:
why rename `_latest_state_tick` to `_latest_state`?
The type declaration is not visible at any line below the declaration, so a user seeing the variable doesn't know if its a dictionary, integer, or a class, unless he scrolls to the top or ctrl+clicks for declaration

Reply:
Wasn't confident in the change, but your tick vs. state note on another comment was convincing. Renamed.

Though I wouldn't base my argument on the type info :slightly_smiling_face: If someone wants to know the type, the variable is statically typed so they can check it from autocomplete, or just look at the definition. The missing info here is not the language type ( int ) itself, but the semantic info that this marks a tick and not a full state.